build/xtask/src/flash.rs

```diff
@@ -78,6 +80,14 @@ impl FlashConfig {
         self
     }
 
+    //
+    // Set the chip
+    //
+    fn chip<'a>(&'a mut self, val: &str) -> &'a mut Self {
```

Comment:
We don't need explicit lifetimes here:
```suggestion
    fn chip(&mut self, val: &str) -> &mut Self {
```

Reply:
(also, maybe name this `set_chip` instead of just `chip`?)

Reply:
Fixed